app/src/main/java/fr/free/nrw/commons/RequestAndNotifyInterface.java

```diff
@@ -0,0 +1,8 @@
+package fr.free.nrw.commons;
+
+public interface RequestAndNotifyInterface {
```

Comment:
This could be renamed to something like CategoryImagesCallback, what do you say ?

Reply:
Sounds good! The hardest part for me was to find a name. I'll address your comments ASAP.